src/applications/edu-benefits/tests/1995/edu-1995-helpers.js

```diff
@@ -92,10 +92,30 @@ function completeStemSelection(client) {
   client.click('input[id="root_isEdithNourseRogersScholarshipNo"]');
 }
 
+function completeStemSelectionFor1995s(client) {
```

Comment:
The description mentions only the form name conditional. Not sure if these pieces are relevant to the description. They look fine but wanted to point them out in case they weren't intended as part of this PR.

Reply:
Yes, the conditional logic is based on the answer to this question, so I updated the existing e2e test to make sure we testing both.